src/components/AccountSettings/AccountSettings.tsx

```diff
@@ -0,0 +1,119 @@
+                            label='Please enter your password'
+                            spellCheck={false}
+                            autoComplete='off'
+                            autoCorrect='off'
+                            autoCapitalize='off'
+                        />
+                        <TextField
+                            id='Disable Account Password Entry Two'
+                            required
+                            fullWidth
+                            variant='outlined'
+                            type='password'
+                            value=''
+                            onChange={() => {}}
+                            label='Please enter your password again to DISABLE your account'
+                            spellCheck={false}
+                            autoComplete='off'
+                            autoCorrect='off'
+                            autoCapitalize='off'
+                        />
+                    </span>
+                ),
+            },
+            {
+                text: 'Delete Account',
+                component: (
+                    <span>
+                        <h1>
+                            Delete Account?
+                            <p>
+                                All of your account information will be erased
+                                from Prytaneum. This action is irreversible.
+                                Please enter your password below twice to
+                                confirm.
+                            </p>
+                        </h1>
+                        <TextField
+                            id='Delete Account Password Entry One'
+                            required
+                            fullWidth
+                            variant='outlined'
+                            type='password'
+                            value=''
+                            onChange={() => {}}
+                            label='Please enter your password'
+                            spellCheck={false}
+                            autoComplete='off'
+                            autoCorrect='off'
+                            autoCapitalize='off'
+                        />
+                        <TextField
+                            id='Delete Account Password Entry Two'
+                            required
+                            fullWidth
+                            variant='outlined'
+                            type='password'
+                            value=''
+                            onChange={() => {}}
+                            label='Please enter your password again to DELETE your account'
+                            spellCheck={false}
+                            autoComplete='off'
+                            autoCorrect='off'
+                            autoCapitalize='off'
+                        />
+                    </span>
+                ),
+            },
+        ],
+    };
+}
```

Comment:
I like the different approach. However, from a maintainability perspective, I think I will ask that we convert this to be more similar to all the other ones we have.  I think it's understandable that having all of the components as close to each other in structure as possible makes the project overall easier to maintain.